locale/de/LC_MESSAGES/django.po

```diff
@@ -482,62 +482,62 @@ msgstr ""
 
 #: liqd_product/apps/partners/models.py:16
 msgid "Short description of your municipality"
-msgstr ""
+msgstr "Kurze Beschreibung Ihrer Kommune"
 
 #: liqd_product/apps/partners/models.py:20
 msgid "Logo"
 msgstr "Logo"
 
 #: liqd_product/apps/partners/models.py:21
 msgid "The Logo representing your municipality"
-msgstr ""
+msgstr "Das Logo Ihrer Kommune"
 
 #: liqd_product/apps/partners/models.py:27
 msgid "Slogan"
-msgstr ""
+msgstr "Slogan"
 
 #: liqd_product/apps/partners/models.py:41
 msgid "Description of your municipality"
-msgstr ""
+msgstr "Beschreibung Ihrer Kommune"
 
 #: liqd_product/apps/partners/templates/partner_about.html:4
 #: liqd_product/apps/partners/templates/partner_about.html:8
 #: liqd_product/apps/partners/templates/partner_base.html:29
 msgid "About us"
-msgstr ""
+msgstr "Über uns"
 
 #: liqd_product/apps/partners/templates/partner_base.html:15
 msgid "Menu"
 msgstr "Menü"
 
 #: liqd_product/apps/partners/templates/partner_landing_page.html:26
 msgid "About this page"
-msgstr ""
+msgstr "Über diese Seite"
 
 #: liqd_product/apps/partners/templates/partner_landing_page.html:31
 msgid "Activity"
-msgstr ""
+msgstr "Aktivitäten"
 
 #: liqd_product/apps/partners/templates/partner_landing_page.html:36
 #: liqd_product/apps/users/templates/liqd_product_users/user_detail.html:108
 msgid "No activity yet. "
-msgstr ""
+msgstr "Bisher fand keine Aktivität statt"
 
 #: liqd_product/apps/partners/templates/partner_landing_page.html:42
 msgid "Statistics"
-msgstr ""
+msgstr "Statistiken"
 
 #: liqd_product/apps/partners/templates/partner_landing_page.html:46
 msgid "registered Users"
-msgstr ""
+msgstr "Angemeldete Nutzer*innen"
 
 #: liqd_product/apps/partners/templates/partner_landing_page.html:50
 msgid "Items"
-msgstr ""
+msgstr "Items"
```

Comment:
I don't think this is a german word. What about "Einträge"?

Reply:
we are using the german "translation" of item in
```
msgid "Item of type %(item)s for that module already exists"
msgstr "Item des Typs %(item)s für dieses Modul existiert bereits."
```
and also beiträge
```
msgid "%(name)s added a new <a href=\"%(item_url)s\">item</a>"
msgstr "%(name)s hat einen neuen <a href=\"%(item_url)s\">Beitrag</a> erstellt"
```

and we will probably drop the statistics block, so i did not put too much thought into it

Reply:
We consistently use "Beitrag" except for the on place you quoted. I would change it there.